rholang/src/main/scala/coop/rchain/rholang/interpreter/compiler/normalizer/processes/PBundleNormalizer.scala

```diff
@@ -28,13 +28,20 @@ object PBundleNormalizer {
     def error(targetResult: ProcVisitOutputs): F[ProcVisitOutputs] = {
       val errMsg = {
         def at(variable: String, sourcePosition: SourcePosition): String =
-          variable + " line: " + sourcePosition.row + ", column: " + sourcePosition.column
+          variable + " at " + sourcePosition.toString
```

Comment:
Scala has nice ways to write string interpolation which looks much nicer and has the same performance.
https://dkomanov.medium.com/scala-string-interpolation-performance-21dc85e83afd#fed5

```scala
s"$variable at $sourcePosition"
```

Strings in Java and C# are reference objects but behave like they are immutable by value objects and sometimes can involve copying on unexpected places like function calls.